cmd/gcrane/cmd/gc.go

```diff
@@ -29,28 +31,57 @@ func init() { Root.AddCommand(NewCmdGc()) }
 // NewCmdGc creates a new cobra.Command for the gc subcommand.
 func NewCmdGc() *cobra.Command {
 	recursive := false
+	before := time.Duration(0)
+	pattern := ""
+
 	cmd := &cobra.Command{
 		Use:   "gc",
 		Short: "List images that are not tagged",
 		Args:  cobra.ExactArgs(1),
 		Run: func(_ *cobra.Command, args []string) {
-			gc(args[0], recursive)
+			gc(args[0], recursive, time.Now().Add(-before), pattern)
 		},
 	}
 
 	cmd.Flags().BoolVarP(&recursive, "recursive", "r", false, "Whether to recurse through repos")
+	cmd.Flags().DurationVarP(&before, "before", "b", time.Duration(0), "Age of the oldest uploaded image to print (ns, us, ms, s, m, h)")
```

Comment:
@ImJasonH what do you think about [borrowing the flag name](https://github.com/sethvargo/gcr-cleaner#payload--parameters) `grace` from @sethvargo instead of `before`?

Reply:
Agree